charts/seed-operatingsystemconfig/original/templates/containerd/containerd-config-generator.service

```diff
@@ -0,0 +1,15 @@
+{{- define "containerd-config-generator" -}}
+- name: containerd-config-generator.service
+  command: start
+  enable: true
+  content: |
+    [Unit]
+    Description=Containerd config generator
+    [Install]
+    WantedBy=multi-user.target
+    [Service]
+    Type=oneshot
+    RemainAfterExit=yes
+    ExecStart=/opt/bin/init-containerd
+{{- end -}}
+
```

Comment:
```suggestion
```

Reply:
this just calls the contanerd init, maybe a better name `containerd-initializer.service`?